research/src/fetch_sex_places.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, company, key=None):
         :param company: (dict) Company with name, cnpj, latitude and longitude
         :param key: (str) Google Places API key
         """
+        global config  # config is define in the installation process
```

Comment:
`config` is not a global variable. It should be imported: `from decouple import config`.